src/aks-preview/HISTORY.rst

```diff
@@ -13,6 +13,10 @@ Pending
 +++++++
 * Vendor new SDK and bump API version to 2023-11-02-preview.
 
+1.0.0b6
```

Comment:
As this is a breaking change (renaming options), following [the new version naming pattern](https://github.com/Azure/azure-cli-extensions/pull/7151#discussion_r1446925040), please choose a version number like `2.0.0b1` and update `VERSION` in `setup.py`.

Reply:
Created new version `2.0.0b1`